execution_workflows/GETUTR/modules/postprocessing_identification.nf

```diff
@@ -0,0 +1,29 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+def modules = params.modules.clone()
+// get the configs for this process
+def options = modules['final_output']
+
+/*
+    Convert GETUTR output file to differential challenge file
+*/
+process POSTPROCESSING_IDENTIFICATION {
+    publishDir "${params.outdir}", mode: params.publish_dir_mode
```

Comment:
Could I suggest, that we write these files:

results/getutr/${sample}_identification_output.bed

To avoid introducing new parameters?

Would that be ok?

Example `results/getutr` content:

```
-rw-r--r-- 1 gregor mering 4296 Oct  4 17:51 SRR6795718_identification_output.bed
-rw-r--r-- 1 gregor mering 3544 Oct  4 17:51 SRR6795719_identification_output.bed
```

Or is there another standard way to do it for APAeval?

Reply:
okay that's fine too! I'm good with results/getutr/${sample}_identification_output.bed!

Reply:
ok cool, done, pushed a new commit now